models/user.go

```diff
@@ -1341,7 +1366,9 @@ type SearchUserOptions struct {
 	UID           int64
 	OrderBy       SearchOrderBy
 	Page          int
-	PageSize      int // Can be smaller than or equal to setting.UI.ExplorePagingNum
+	Private       bool  // Include private orgs in search
```

Comment:
VisibleType?

Reply:
I used `Private` to match the options in `SearchRepoOptions`

Reply:
Hmm I think we're going to immediately get an issue opened with a user complaining that they want to search on the different types of visibility - is it really disruptive to change it or do you think we should deal with that when the inevitable occurs?

Reply:
I think we can add this in later with a slice for VisibleTypes and have a conditional (after the other conditionals) that says `visibility in ([]VisibilityTypes)`, that way it'll filter on the already filtered list to ensure that a org that the user can't see isn't accidentally shown to user.

I am hesitant to add this in now, and want to get the minimal feature set out for this before adding anything else.